src/Archipelago/ArchipelagoIntegration.cs

```diff
@@ -397,6 +415,80 @@ public void SendDeathLink() {
             deathLinkService.SendDeathLink(new DeathLink(Player, $"{Player}{DeathLinkMessages.Causes[AreaDiedIn][new System.Random().Next(DeathLinkMessages.Causes[AreaDiedIn].Count)]}"));
         }
 
+
+        public void EnableTrapLink() {
+            if (!session.ConnectionInfo.Tags.Contains("TrapLink")) {
+                session.ConnectionInfo.UpdateConnectionOptions(session.ConnectionInfo.Tags.Concat(new string[1] { "TrapLink" }).ToArray());
+            }
+        }
+
+        public void DisableTrapLink() {
+            if (session.ConnectionInfo.Tags.Contains("TrapLink")) {
+                session.ConnectionInfo.UpdateConnectionOptions(session.ConnectionInfo.Tags.Where(tag => tag != "DeathLink").ToArray());
```

Comment:
```suggestion
                session.ConnectionInfo.UpdateConnectionOptions(session.ConnectionInfo.Tags.Where(tag => tag != "TrapLink").ToArray());
```
I'm assuming this should be TrapLink